### ISSUE TITLE
GONG - konečný stav

### ISSUE THREAD

**Issue by Yendy** [Yendy/petkari.cz#30]
dtto. KRONIKA
+
ročníky.

**Comment by kalich5** [comment]
Obecně bych to rozšířil na všechny časopisy. Zkusím zjistit, kolik jich třeba za oddílovou historii bylo

**Comment by Yendy** [comment]
Ano, můžeme to aplikovat obecně, tj. nebude pouze časopis GONG, ale bude typ Časopis > Gong, Pětník atd.

**Comment by kalich5** [comment]
To by bylo nejlepší, táta bude přes Velikonoce v Čechách, tak to s ním proberu.
